generators/app/templates/src/scss/application.scss

```diff
@@ -1,21 +1,21 @@
 @import '~wolox-equalizer/equalizer';
 
 @import 'layout';
+
```

Comment:
are these newlines necessary?

Reply:
Yes sir!. Otherwise our stylelint is gonna say:
![tenor-253082678](https://user-images.githubusercontent.com/36163179/56812491-60d7d100-6811-11e9-9599-d4c4435f3e86.gif)

Reply:
<img width="941" alt="Captura de Pantalla 2019-04-26 a la(s) 10 53 35" src="https://user-images.githubusercontent.com/36163179/56812747-e3f92700-6811-11e9-8acd-9263e6a63862.png">